Lib/test/test_socket.py

```diff
@@ -1217,8 +1220,8 @@ def testGetServBy(self):
         else:
             raise OSError
         # Try same call with optional protocol omitted
-        # Issue #26936: Android getservbyname() was broken before API 23.
-        if (not support.is_android) or sys.getandroidapilevel() >= 23:
+        # Issue #26936: this fails on Android before API level 23.
```

Comment:
Same suggestion below:

```suggestion
        # Issue gh-71123: this fails on Android before API level 23.
```